lib/Dredd.js

```diff
@@ -49,6 +49,7 @@ class Dredd {
     };
     this.transactions = [];
     this.runner = new Runner(this.configuration);
+    this.logger = logger;
```

Comment:
Not a _great_ solution, but will do for now. I want to refactor this part of the codebase further in future PRs.